node/http.ts

```diff
@@ -337,7 +337,7 @@ export class ServerResponse extends NodeWritable {
     const body = singleChunk ?? (final ? null : this.#readable);
     if (this.#isFlashRequest) {
       this.#resolve!(
-        new Response(Buffer.isBuffer(body) ? body.toString() : body, {
```

Comment:
`body.toString()` is unnecessary (because a Buffer is a Uint8Array) and also problematic when body contains non-text content.